# RNA-seq Analysis Pipeline

## Overview
This proyect implements an RNA- seq analysis workflow using publicly available data.
The aim is to identify differentially expressed genes between experimental conditions.

## Dataset
The RNA-seq data was obtained from a public GEO dataset. (Exact accession number to be added.)

## Workflow
1. Quality control of raw reads.
2. Read alignment / quantification.
3. Gene-level expression analysis.
4. Differential expression analysis.
5. Visualization of results.

## Tools
- Bash
- Python
- R
- Snakemake (¿?)

## Status
Searching the dataset for the experiment.

## Dataset
**GEO:** GSE266108\
**Link:** [GEO Dataset](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE266108)\
**Summary:** This study investigates the role of sodium butyrate (NaB) in diabetic kidney disease (DKD), a condition characterized by impaired lipid metabolism and defective lipophagy. Using glucolipotoxicity (GLT)-stimulated human renal tubular epithelial cells (HK-2), RNA-seq was performed to compare gene expression profiles with and without NaB treatment. The results show that NaB improves lipid accumulation and renal dysfunction by restoring lipophagy, likely through activation of the PP2A–TFEB signaling pathway. This dataset enables the analysis of differential gene expression associated with NaB treatment in a controlled cellular model of DKD.\
**Overall design:** This RNA-seq dataset compares GLT-stimulated HK2 renal cells with and with sodium butirate versus untreated control.


## Research Question
Which genes are differentially expressed between two experimental conditions in an RNA-seq experiemnt using public data?

This project focuses on applying a standard RNA-seq workflow to answer a biologically relevant question in a reproducible way.

## VS Studio
This last line is edited and updated exclusively from VS Studio. It´s a try out.

